spec/ext/vixDiskLib.c

```diff
@@ -0,0 +1,476 @@
+#include <stdlib.h>
+#include <stddef.h>
+#include <stdarg.h>
+#include <stdint.h>
+
+typedef uint64_t uint64;
+typedef  int64_t  int64;
+typedef uint32_t uint32;
+typedef  int32_t  int32;
+typedef uint16_t uint16;
+typedef  int16_t  int16;
+typedef  uint8_t  uint8;
+typedef   int8_t   int8;
+typedef     char   Bool;
+typedef uint64 VixError;
+
+enum {
+   VIX_OK = 0,
+};
+
+typedef struct VixDiskLibHandleStruct VixDiskLibHandleStruct;
+typedef VixDiskLibHandleStruct *VixDiskLibHandle;
+typedef void (VixDiskLibGenericLogFunc)(const char *fmt, va_list args);
+struct VixDiskLibConnectParam;
+typedef struct VixDiskLibConnectParam *VixDiskLibConnection;
+typedef Bool (*VixDiskLibProgressFunc)(void *progressData,
+                                       int percentCompleted);
+typedef void (*VixDiskLibCompletionCB)(void *cbData, VixError result);
+typedef enum {
+   VIXDISKLIB_DISK_MONOLITHIC_SPARSE         = 1,   // monolithic file, sparse
```

Comment:
Do we need the actual enum value?  Thinking we can drop those too

Reply:
An empty enum is a compilation error, but we could leave just the `UNKNOWN = 256` in each of these.
Alternatively we could change the function prototypes and structs to take an int instead of an enum and delete all the enums.